beacon_node/lighthouse_network/src/config.rs

```diff
@@ -352,6 +356,7 @@ impl Default for Config {
             outbound_rate_limiter_config: None,
             invalid_block_storage: None,
             inbound_rate_limiter_config: None,
+            idontwant_message_size_threshold: 1000,
```

Comment:
It'd be good to extract the default value into a `IDONTWANT_MESSAGE_SIZE_THRESHOLD` const value.

Reply:
sounds reasonable. made the requested changes

Reply:
`target_peers` few line above also seems to have inline declaration should I extract it too?